src/prommis/uky/autoscaling.py

```diff
@@ -0,0 +1,193 @@
+from pyomo.environ import (
```

Comment:
@andrewlee94 Is it fine to merge the autoscaling file in this repo? I think at the very least, we could move it out of the uky folder since its meant for general use, not just this one application. Also, was this autoscaling tool intended to be in the IDAES repo eventually (or maybe it's already there and I just need to update the import in the uky flowsheet)?

Reply:
The autoscaling probably should not make it in, as it will eventually end up in IDAES as a utility tool.

Reply:
The problem with not merging it in is that the flowsheet will almost definitely fail without it.

Reply:
Could you grab the Suffix it creates and just add that manually?

Reply:
Not sure what you mean by that or how to do it

Reply:
You can display the Suffix using `model.scaling_factor.display()`. All the scaling factors are stored in that one Suffix (basically a dict, where the keys are components and ht value the scaling factor).

Reply:
The flowsheet actually solves the same if I just remove the autoscaling, so I think my manual scaling may have already been very similar to the autoscaling

Reply:
@lbianchi-lbl How would you like to handle this? Basically the following changes should be made:

1.) The autoscaling file should be removed

2.) The autoscaling import should be removed from the uky_flowsheet.py

3.) The solve function in the uky_flowsheet.py should look like this
`def solve(m):
    solver = SolverFactory("ipopt")
    results = solver.solve(m, tee=True)

    return results`

Reply:
Thanks @MarcusHolly. I'm not yet sure if I'll do this as part of the Git history migration happening momentarily, or immediately afterwards. I've created #7 to track this.